bridges/python/Pipfile

```diff
@@ -8,5 +8,7 @@ requests = "==2.21.0"
 pytube = "==9.5.0"
 tinydb = "==3.9.0"
 beautifulsoup4 = "==4.7.1"
+pyowm = "==2.10.0"
```

Comment:
This PR is quite old, before we merge, we should use the latest versions of the dependencies used in this package.
Currently the latest version of `pyowm` is `3.2.0` and `tzlocal` is `2.1`.